Sources/Rendering/Core/Renderer/index.js

```diff
@@ -597,7 +597,7 @@ export function extend(publicAPI, model, initialValues = {}) {
     'pass',
   ]);
   macro.getArray(publicAPI, model, ['actors', 'volumes', 'lights']);
-  macro.setGetArray(publicAPI, model, ['background'], 3);
+  macro.setGetArray(publicAPI, model, ['background'], [3, 4]);
```

Comment:
y, I updated macro.js to handle it, I think....

Adding a default value for shorter arrays makes sense to me, and could be useful in other situations. If a default value is not supplied, then I'll still throw an error if the array is too short.